descriptors/ModuleDescriptor-template.json

```diff
@@ -102,7 +102,7 @@
           ],
           "pathPattern": "/settings/upload",
           "permissionsRequired": [
-            "mod-settings.entries.upload"
```

Comment:
You didn't do `replaces` for this. That's really not on, as it makes this a backwards-incompatible change. Please refer to the commits @nielserik created when making analogous changes in other modules.